src/rules/comment-no-loud/__tests__/index.js

```diff
@@ -31,6 +31,21 @@ testRule(rule, {
     `,
       description: "Multline comment with optional ending on next line",
       message: messages.expected
+    },
+    {
+      code: `
+      i {
+        font-style: italic;
+      }
+
+      /* comment line */
+
+      b {
+        font-weight: bold;
+      }
+    `,
+      description: "Comment sandwiched between rules",
+      message: messages.expected
```

Comment:
I noticed that the existing tests are missing the `line` and `column` numbers from the rejected test cases. Could you please add those to your test?

Reply:
@kristerkari Sure, I've added those in now.